src/internal/operators/skipUntil.ts

```diff
@@ -10,8 +10,35 @@ import { Subscription } from '../Subscription';
 /**
  * Returns an Observable that skips items emitted by the source Observable until a second Observable emits an item.
  *
+ * The `skipUntil` operator causes the observable stream to skip the emission of values ​​until the passed in observable returns the first value.
```

Comment:
`observable returns the first value.` I think it would be clearer if it says `observable emits the first value`

Reply:
Thanks @JWO719. I've implemented your requested change.